lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -1878,19 +1843,14 @@ export class UserAgentApplication {
             if (!Utils.isEmpty(authority)) {
               authority = Utils.replaceTenantPath(authority, response.idToken.tenantId);
             }
-            if (response.idToken.objectId) {
-              response.uniqueId = response.idToken.objectId;
```

Comment:
why removed?

Reply:
This was not removed. This was extracted into the responseWithIdToken function.